src/data/MetaDataItem.py

```diff
@@ -75,6 +83,10 @@ def get_tag(self, name: str):
         else:
             return None
 
+    def clone(self):
+        return MetaDataItem(**self.to_json())
```

Comment:
honestly we should probably just set the id to none here so that clone never creates an exact duplicate metadataitem